testing/src/test/java/io/stargate/it/CQLTest.java

```diff
@@ -182,14 +182,7 @@ public void querySystemLocal() throws UnknownHostException {
     ResultSet rs = session.execute("SELECT * FROM system.local");
     Iterator<Row> rows = rs.iterator();
     assertThat(rows).hasNext();
-    assertThat(rows.next().getInetAddress("listen_address"))
-        .isEqualTo(InetAddress.getByName(stargateHost));
-  }
-
-  @Test
-  public void querySystemPeers() {
-    ResultSet rs = session.execute("SELECT * FROM system.peers");
-    assertThat(rs.all()).isEmpty();
+    assertThat(rows.next().getInetAddress("listen_address")).isIn(getStargateInetSocketAddresses());
```

Comment:
Should we validate all rows to be an exhaustive list of configured addresses?

Reply:
system local can contain any address from there, the request can go to any node because of the `LoadBalancingPolicy` so such a test would be more fragile

Reply:
Also, this test case seems to be a covered by `MultipleStargateInstancesTest`. Why would we have both?

Reply:
but doesn't `system.peers` reflect all Stargate nodes?

Reply:
oops, sorry, I got confused by the diff - it does apply to `system.local` now - please ignore